core/experiments/manager.php

```diff
@@ -528,32 +529,61 @@ private function render_settings_intro() {
 	 * @param array $feature
 	 */
 	private function render_feature_settings_field( array $feature ) {
+		$control_id = 'e-experiment-' . $feature['name'];
```

Comment:
Seems like this feature is getting bigger and bigger and I think we should consider recreating this view with react and using REST instead of the `initial_setting`